ranger-drove-client/src/main/java/io/appform/ranger/client/drove/SimpleRangerDroveClient.java

```diff
@@ -42,7 +44,8 @@ public class SimpleRangerDroveClient<T> extends AbstractRangerClient<T, ListBase
     private final DroveResponseDataDeserializer<T> deserializer;
     @Builder.Default
     private final ShardSelector<T, ListBasedServiceRegistry<T>> shardSelector = new ListShardSelector<>();
-
+    @Builder.Default
+    private final ServiceNodeSelector<T> nodeSelector = new RandomServiceNodeSelector<>();
```

Comment:
I do not see, how we are planning to propagate weights etc in case we are using drove based service discovery?